### ISSUE TITLE
Generation failure

### ISSUE THREAD

**Issue by Bridouz** [mosra/m.css#2]
Hi,

Wanted to try your theme (which I find gorgeous) on a plain Pelican installation.

I followed your guide to add theme files on Pelican,
i even copied your template *rst* article

And when I type `make devserver` I get this

```
 File "/home/justin/Pelican/Blog/m.css/pelican-theme/templates/article_header.html", line 4, in top-level template code
    {% set month, day, year = article.locale_date.split(' ') %}
ValueError: too many values to unpack (expected 3)
```

**Comment by mosra** [comment]
Hey! Thanks for the report!

I'm very sorry for replying so late, missed this completely (why is github not sending notification e-mails for new issues? ugh).

This is an omission in the docs on my end (I have a fix in the works). Currently, in order to unbreak it, you need to specify additional settings in `pelicanconf.py` for article dates so it formats the date in the way that the template expects it:

```py
DEFAULT_LANG = 'en'
DATE_FORMATS = {'en': ('en_US', '%b %d %Y')}
```

This should be hopefully everything needed.

Keeping this open until I update the docs (or fix it in some better way).

**Comment by mosra** [comment]
I pushed a fix in c13616362d526cd4a924a7e4bcc4432f44dce119 -- the `DATE_FORMATS` setting no longer needs to be set to any specific value, only the locale setting from it affects how the dates will look.

Closing as resolved, please reopen if you still have the problem :)
